### PR TITLE
python/update-to-book-version

### PR DESCRIPTION
**Pull request by christophscheuch**

### CODE REVIEW

File: .Rprofile [hunk 1]
```diff
@@ -1,9 +1,2 @@
 source("renv/activate.R")
 source("r/pre_render_script.R")
```
Comment (voigtstefan) [inline .Rprofile:2]: Remove source("r/pre_render_script.R"). There is no reason to call the script by default for every session.

File: bib/book.bib [hunk 2]
```diff
@@ -694,6 +724,13 @@ @article{Hautsch2019
 	pages        = {221--240},
 	url          = {https://doi.org/10.1016/j.jeconom.2019.04.028},
 }
+@book{Hilpisch2018,
+  title 		 = {Python for Finance},
+  author 		 = {Hilpisch, Yves},
+  year 			 = {2018},
+  publisher 	 = {O'Reilly Media, Inc.},
+  edition 	  	 = {second},
```
Comment (voigtstefan) [inline bib/book.bib:732]: url = {https://www.oreilly.com/library/view/python-for-finance/9781492024323/}

**Review by voigtstefan — COMMENTED** [review]
Thanks for transferring the files. I left two small comments but did not touch the code myself - your call if you want to include these points.

File: .Rprofile (hunk 1 above)
Comment (christophscheuch) [reply .Rprofile:2]: I think the idea was to have the same figure specs in your RStudio interactive session as in the render, but I'm ok with deleting it.

**Pull request merged**